x-pack/plugins/siem/cypress/objects/rule.ts

```diff
@@ -4,7 +4,12 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export const totalNumberOfPrebuiltRules = 127;
+// eslint-disable-next-line
+import { rawRules } from '../../server/lib/detection_engine/rules/prepackaged_rules/index';
+
+export const totalNumberOfPrebuiltRules = rawRules.length;
+
+export const totalLoadedRules = 130;
```

Comment:
Nit: it might be useful to keep the `prebuilt` qualifier in this constant, as it's a little vague right now. For posterity: this is the number of rules that exists in the `prebuilt_rules_loaded` archive.

Reply:
Lots of thanks for the suggestion, you are completely right 😄